resources/Login.py

```diff
@@ -0,0 +1,58 @@
+from flask import Flask, jsonify, make_response
+from flask_restful import Resource, reqparse
+from firebase_admin import credentials, firestore
+
+app = Flask(__name__)
+
+# Initialize Firestore DB
+db = firestore.client()
+users_ref = db.collection('users')
+
+
+class Login(Resource):
+    def get(self):
+        return {'username': 'username',
+                'password': 'password'}, 200
+
+    def post(self):
+        # parse args
+        login_post_args = reqparse.RequestParser()
+        login_post_args.add_argument("username", type=str, help="username is required", required=True)
+        login_post_args.add_argument("password", type=str, help="password is required", required=True)
+        args = login_post_args.parse_args()
+
+        username = args['username'].lower()
+        password = args['password']
+        try: 
+            doc = users_ref.document(username).get()
+            if not doc.exists:
+                return "user does not exist", 401
```

Comment:
Return the same message. Just say login failed. You are exposing private info like this